iocore/net/SSLSNIConfig.cc

```diff
@@ -93,6 +97,78 @@ NamedElement::set_regex_name(const std::string &regex_name)
 ////
 // SNIConfigParams
 //
+// ----
+
+SNI_IpAllow::SNI_IpAllow(std::string &ip_allow_list, std::string const &servername)
+{
+  ts::TextView content{ip_allow_list};
+  if (content && content[0] == '@') {
+    std::error_code ec;
+    ts::file::path path{content.remove_prefix(1)};
+    ts::LocalBufferWriter<1024> w;
```

Comment:
This is only used when error occurs.

Reply:
This is now moved to SNIActionPerformer.cc. Addressed in the other conversation.